Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/BooleanSetting.java

```diff
@@ -39,6 +39,7 @@ public enum BooleanSetting implements AbstractBooleanSetting
           true),
   MAIN_ENABLE_SAVESTATES(Settings.FILE_DOLPHIN, Settings.SECTION_INI_CORE, "EnableSaveStates",
           false),
+  MAIN_BBA_XLINK_CHAT_OSD(Settings.FILE_DOLPHIN, Settings.SECTION_INI_CORE, "BBA_XLINK_CHAT_OSD", false),
```

Comment:
Please order the settings in the same order as in C++. (Same in StringSetting.java.)

Reply:
i tried to order this the same way as MainSettings.h

Reply:
Well, it isn't in the same order. `MAIN_BBA_XLINK_CHAT_OSD` comes way earlier than `MAIN_ENABLE_SAVESTATES` in MainSettings.h.

Reply:
Oh, nevermind, you meant that in the latest commit it's ordered the same way. Yeah, it seems reasonable now.